app/[locale]/(farmer)/dashboard/@recent/page.tsx

```diff
@@ -17,9 +19,12 @@ export default async function RecentEntries() {
 
   return <div>
     {journals.map(journal => {
-      return <div key={journal.id} className="py-2 px-6 my-1 bg-base-200">
-        <p>Date: {enFullDate.format(new Date(journal.created))}</p>
-        <p>Chicken: {journal.fieldInitialstock || 0}</p>
+      return <div key={journal.id} className="py-4 px-6 my-1 bg-base-200">
+        <p>{formatter.dateTime(new Date(journal.created), {
+          day: "numeric",
+          month: "long",
+          year: "numeric"
+        })}</p>
```

Comment:
Could we make a utility function to handle these cases? Or is it not worth it?  (see below)

Reply:
The `formatter` uses the Intl API and (almost) has everything we need for i18n, so no need to write a separate function for that.